opds_import.py

```diff
@@ -402,9 +403,10 @@ class OPDSImporter(object):
     SUCCESS_STATUS_CODES = None
 
     def __init__(self, _db, collection, data_source_name=None,
-                 identifier_mapping=None, mirror=None, http_get=None,
+                 identifier_mapping=None, http_get=None,
                  metadata_client=None, content_modifier=None,
-                 map_from_collection=None,
+                 map_from_collection=None, covers_mirror=None,
+                 books_mirror=None,
```

Comment:
In ReplacementPolicy you have a dictionary of mirrors keyed by purpose, and I think it's best to do the same here. Not only will this make everything consistent, this will let us add other kinds of mirrors (such as for book previews) without changing the method signature.